pytest.ini

```diff
@@ -1,2 +1,2 @@
 [pytest]
-addopts = tests
+#addopts = tests
```

Comment:
Are we getting rid of this? If yes, I'd change the documentation and possibly setup.py test command to include "tests", perhaps.

Reply:
No, I committed it accidentally.